proto/catalog.proto

```diff
@@ -246,6 +269,7 @@ message Connection {
   string name = 4;
   oneof info {
```

Comment:
No need to keep the `oneof` now. May remove it to improve readability.

Reply:
We are still using the proto in meta backup. Removing the `oneof` might break compatibility.